Localization/de-DE/about.yaml

```
@@ -0,0 +1,12 @@
+---
+url: "https://foldingathome.org/about/"
+language: "de-DE"
+title: "ÜBER"
+text: [
+# Bold text at the start
+"Folding@home (FAH oder F@h)",
+
+# The rest of the paragraph
+#"is a distributed computing project for simulating protein dynamics, including the process of protein folding and the movements of proteins implicated in a variety of diseases. It brings together citizen scientists who volunteer to run simulations of protein dynamics on their personal computers. Insights from this data are helping scientists to better understand biology, and providing new opportunities for developing therapeutics."
+"ist ein Projekt für verteiltes Rechnen zur Simulation von Proteindynamiken, einschließlich des Prozesses der Proteinfaltung und der Bewegung von Proteinen, die innerhalb verschiedenster Erkrankungen eine wichtige Rolle spielen. Es bringt Amateurwissenschaftler zusammen, die sich freiwillig melden um Simulationen dieser Proteindynamik auf ihrem PC laufen zu lassen. Wissenschaftler können durch diese Daten neue Einblicke gewinnen, um Biologie besser zu verstehen und neue Möglichkeiten zur Entwicklung von Therapeutika zu entwickeln."
```

```suggestion
"ist ein Projekt für verteiltes Rechnen zur Simulation von Proteindynamiken, einschließlich des Prozesses der Proteinfaltung und der Bewegung von Proteinen, welche innerhalb verschiedenster Erkrankungen eine wichtige Rolle spielen. Es bringt Menschen zusammen welche freiwillig Simulationen dieser Proteindynamik auf ihren PCs laufen lassen. Wissenschaftler können durch diese Daten neue Einblicke gewinnen, um diese Vorgänge besser zu verstehen und neue Möglichkeiten zur Entwicklung von Therapien zu entwickeln."
```
Replaced 

- Replaced ", die" mit ",welche"
- "Amateurwissenschaftler" sounds a little bit strange to me in this context and may scare off people
- "Biologie" is a bit to wide spread in my opinion
- "Therapien" is more common/general than "Therapeutika"

---

That again raises the question how much liberty we should take or if it would be better to stick as closely to the original as possible. I'm in favor of doing the latter and only making changes if the result doesn't sound like a normal German sentence at all.

Making editorial changes would require a check-in with the core team (e.g. the original author) in my opinion.

---

To which change are you exactly referring? Imho German  is always hard to translate from English cause German is much more descriptive than English and same words directly translated are sometimes completely different. As the general translation sense is the same and the whole webpage is like marketing to get the people folding imho the changes will reflect it better. But i completely agree that this is something to discuss in a wider scope!

---

die -> welche: agree, will do
amatuerwissenschaftler: agree, used it for lack of a better word.
biologie: what do you propose as an alternative?
therapeutika -> therapien: they are strictly speaking bout therapeutika though, aren't they? thats why i chose that word.

---

@dsdel Oh yeah...I kind of focused on only one thing. I mean changing "Amateurwissenschaftler" and "Biologie". Both are kind of pretty clearly stated in the original. Changing that to entirely different words feels not quite right for me. And of course literal translations are not a good idea. But it's definitely a fine balance between preserving the tone/meaning of the original text and making it sound natural in German.

---

- _Biologie_, then let us keep it : )
- _Amateurwissenschaftler_ - I for myself would really like to change it. Citizen scientists and _Amateurwissenschaftler_ have imho a completely different scope in the languages. As in german scientists is much more strict to the profession and nobody would also ever use some word like _Amateurdoktor_
- _Therapeutika_ - they use the word 'therapeutics' together with 'developing' which is imho a different meaning than _Therapeutika_. Furthermore _Therapeutika_ is very seldom used nowadays. One some other translation we used _Heilmittel_. Perhaps we should better stick using the same translation here also?

---

ok, heilmittel sounds good.
how about just changing amateurwissenschaftler to freiwillige? it fits in the sentence and doesnt seem that weirdly specific.

---

I pondered as well about this "citizen scientist" and how to translate it. Literally it would be a "Bürgerwissenschaftler", but I doubt anyone would call herself like this. So I thought about an astronomer, where citizen astronomers are quite common and they are called "Amateurastronom", as Amateur means they don't do it professionally, as opposed to the "Profi". Me too, I wasn't 100% happy but couldn't come up with something better.
"Freiwillige" doesn't nail it either. Maybe "Hilfswissenschaftler"? Otherwise I'd rather chose the totally neutral "Menschen" over "Freiwillige".

---

i think that just in general in german anything-wissenschaftler seems pretty involved/something very specific where people are very interested in. just using the term wissenschaftler might scare people off installing fah.
Menschen is probably our best option. ill do that and push a new commit.